flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -881,24 +883,21 @@ class GroupWindowedTable(
     * }}}
     */
   def select(fields: Expression*): Table = {
```

Comment:
At the moment there is no way to specify watermarks inside of a Table API or SQL query. This can only be done on a DataStream before it is converted into a Table. Therefore, watermarks and timestamps are already assigned before the first Table or SQL operator can remove anything. In case of a TableSource which assigns timestamps, the TableSource needs to take care that the assignment happens before a pushed-down projection is applied.